hv-rhel6.x/hv/rhel6-hv-driver-install

```diff
@@ -21,15 +21,20 @@ echo "Installing Modprobe Rules"
 cp -f ./hyperv_pvdrivers.conf /etc/modprobe.d/
 [ $? -eq 0 ] || exit 1
 
-echo "Copying scipts for ip injection"
+echo "Copying scripts for IP injection"
 
-cp -f ./tools/hv_get_dns_info.sh /usr/sbin/
+\cp ./tools/hv_get_dns_info.sh /usr/sbin/hv_get_dns_info
```

Comment:
The leading backslash on the cp line looks to be a syntax error, or I'm not familiar with this syntax.  Can you confirm?